ingestors/drupal.go

```diff
@@ -0,0 +1,120 @@
+package ingestors
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/PuerkitoBio/goquery"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/librariesio/depper/data"
+	"github.com/mmcdole/gofeed"
+)
+
+const drupalSchedule = "*/5 * * * *"
+const drupalModulesUrl = "https://www.drupal.org/project/project_module?page=%d&solrsort=ds_project_latest_release+desc"
+const drupalReleasesUrl = "https://www.drupal.org/node/%s/release/feed"
+
+type Drupal struct {
+	LatestRun time.Time
+}
+
+func NewDrupal() *Drupal {
+	return &Drupal{}
+}
+
+func (ingestor *Drupal) Schedule() string {
+	return drupalSchedule
+}
+
+func (ingestor *Drupal) Name() string {
+	return "drupal"
+}
+
+func (ingestor *Drupal) Ingest() []data.PackageVersion {
+	var results []data.PackageVersion
+
+	// Until we save LatestRun state, we need to set a LatestRun to avoid scanning every single release in the index.
+	if ingestor.LatestRun.IsZero() {
+		ingestor.LatestRun = time.Now().Add(-30 * 24 * time.Hour)
+	}
```

Comment:
I honestly can't remember the difference between the bookmark times and `LatestRun`, so I just went with the latter.

Reply:
Also, not sure how far we go back on the first run  ... is 30 days enough?

Reply:
Note to self: look at the oldest package in the manifest for Catalyst on Monday, to determine how far we need to go back.

Reply:
> I honestly can't remember the difference between the bookmark times and `LatestRun`, so I just went with the latter.

The difference primarily is that bookmarks are persistent between runs.

Reply:
😆😆😆 thanks @jsonperl !! We miss youuuuu

Reply:
Miss y’all too!